web/src/pages/Proto/components/ProtoDrawer/index.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useEffect } from 'react';
+import { Button, Drawer, Form, notification, Space } from 'antd';
+import { useIntl } from 'umi';
+import Editor from '@monaco-editor/react';
+import { Input } from 'antd';
+import { create, update } from '../../service';
+import styles from './index.less';
+
+const ProtoDrawer: React.FC<ProtoModule.ProtoDrawerProps> = ({
+  protoData,
+  setProtoData,
+  visible,
+  setVisible,
+  editMode,
+  refreshTable,
+}) => {
+  const [form] = Form.useForm();
+  const { formatMessage } = useIntl();
+
+  useEffect(() => {
+    form.setFieldsValue(protoData);
+  }, [visible]);
```

Comment:
Is it possible to use `initialValues` API from `Form`?

Reply:
This operation is required, I need to guarantee that the form data is reset when the drawer is opened.